Jenkinsfile-auto-deploy-marker

```diff
@@ -0,0 +1,32 @@
+def issueUrl = "https://api.github.com/repos/Wikia/design-system/issues/${params.pull_num}"
+def authHeader = "-H 'Authorization: token ${GITHUB_TOKEN}'"
+def contentTypeHeader = "-H 'Content-type: application/json'"
+def labels = "${params.labels}"
+
+node('linux') {
+	try {
+		stage('Add label to GitHub repo') {
+			def githubUsername = "WikiaJenkins"
+
+			def publishMinorLabel = "publish-minor-on-merge"
+			def deployDocsLabel = "deploy-docs-on-merge"
+
+			if (!labels.contains(publishMinorLabel)) {
+				sh "curl -X POST ${authHeader} ${contentTypeHeader} --data '[ \"${publishMinorLabel}\" ]' ${issueUrl}/labels"
```

Comment:
Is there a need for this curl here? Labels are already provided in the `labels` parameter and they should come with the webhook event

Reply:
This is a POST request, so it'll change the labels.